### PR TITLE
Update packages and fix lints

### PR DESCRIPTION
**Pull request by manzoorwanijk**
This PR updates all packages to the latest version. Due to this, some new rules have been added to TS Lint plugin. It created a lot of lints, this PR addresses those lints.

### CODE REVIEW

File: assets/src/application/services/theme/theme.ts [hunk 1]
```diff
@@ -2,17 +2,13 @@ import { theme as chakraTheme, DefaultTheme } from '@chakra-ui/core';
 
 type CustomTheme = DefaultTheme;
 
-const breakpoints = ['30em', '48.875em', '62em', '80em'];
+const breakpoints: any = ['30em', '48.875em', '62em', '80em'];
```
Comment (nicolad) [inline assets/src/application/services/theme/theme.ts:5]: easy

File: assets/src/domain/shared/entities/prices/predicates/selectionPredicates/index.ts [hunk 2]
```diff
@@ -7,10 +6,10 @@ import { PRICE_FIELDS, PRICE_INPUT_FIELDS } from '../priceFields';
 import { findEntityByDbId, findEntityByGuid } from '../../../../services/predicates';
 
 // the following return `true` if price satisfies predicate
-export const isPriceField = (_, field): boolean => includes(field, PRICE_FIELDS);
+export const isPriceField = (_: unknown, field: string): boolean => includes(field, PRICE_FIELDS);
 
 // the following return `true` if price satisfies predicate
-export const isPriceInputField = (_, field): boolean => includes(field, PRICE_INPUT_FIELDS);
+export const isPriceInputField = (_: unknown, field: string): boolean => includes(field, PRICE_INPUT_FIELDS);
```
Comment (nicolad) [inline assets/src/domain/shared/entities/prices/predicates/selectionPredicates/index.ts:12]: :+1:

File: package.json [hunk 3]
```diff
@@ -66,59 +66,59 @@
...
 		"@types/graphql": "^14.5.0",
 		"@types/invariant": "^2.2.33",
 		"@types/jest": "^25.2.3",
 		"@types/jquery": "^3.3.38",
-		"@types/node": "^14.0.4",
+		"@types/node": "^14.0.5",
 		"@types/ramda": "^0.27.6",
 		"@types/react": "^16.9.35",
-		"@types/react-beautiful-dnd": "^12.1.2",
+		"@types/react-beautiful-dnd": "^13.0.0",
```
Comment (nicolad) [inline package.json:121]: is this introducing any breaking changes?
since it is a major version upgrade

Comment (nicolad) [reply package.json:121]: :facepalm: it's types-related

File: package.json [hunk 4]
```diff
@@ -157,21 +157,21 @@
...
 		"eslint-plugin-jsx-a11y": "^6.2.3",
 		"eslint-plugin-prettier": "^3.1.3",
 		"eslint-plugin-react": "^7.20.0",
-		"eslint-plugin-react-hooks": "^4.0.2",
+		"eslint-plugin-react-hooks": "^4.0.4",
 		"file-loader": "6.0.0",
 		"fs-extra": "^9.0.0",
 		"graphql": "^15.0.0",
-		"graphql-tools": "^5.0.0",
+		"graphql-tools": "^6.0.3",
```
[the remark this answers is not in the document]
Comment (manzoorwanijk) [reply package.json:174]: No idea, may be some build tools use it?

**Pull request merged**